unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,32 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
```

Comment:
These are good happy path tests, but it would be good to also test the edge cases.